include/bluetooth/conn.h

```diff
@@ -413,6 +413,59 @@ void bt_conn_cb_register(struct bt_conn_cb *cb);
  */
 void bt_set_bondable(bool enable);
 
+/** Allow/disallow remote OOB data to be used for pairing.
+ *
+ *  Set/clear the OOB data flag for SMP Pairing Request/Response data.
+ *  The initial value of this flag depends on BT_OOB_DATA_PRESENT Kconfig
+ *  setting.
+ *
+ *  @param enable Value allowing/disallowing remote OOB data.
+ */
+void bt_set_oob_data_flag(bool enable);
+
+/**
+ * @brief Set OOB data during LE SC pairing procedure
+ *
+ * This function allows to set OOB data during the LE SC pairing procedure. The
+ * function should only be called in response to the oob_data_request() callback
+ * provided that LE SC method is used for pairing.
+ *
+ * The user should submit OOB data according to the information received in the
+ * callback. This may yield three different configurations: with only local OOB
+ * data present, with only remote OOB data present or with both local and
+ * remote OOB data present.
+ *
+ * @param conn Connection object
+ * @param oobd_local Local OOB data or NULL if not present
+ * @param oobd_remote Remote OOB data or NULL if not present
+ *
+ *  @return Zero on success or error code otherwise, positive in case
+ *  of protocol error or negative (POSIX) in case of stack internal error
+ */
+int bt_le_sc_oob_data_set(struct bt_conn *conn,
```

Comment:
I wonder if this should be `bt_le_sc_{set|get}_oob_data` for consistency with (most) other APIs. @jhedberg ?

Reply:
Well, as I mentioned before there is this ```bt_passkey_set``` function, so I think that general refactor is required to make the code aligned with either convention.

However, I can change the oob set/get functions in this PR if you think it's necessary.

Reply:
@carlescufi @jhedberg If we want to make it consistent should it be `bt_le_oob` as the prefix, same as `bt_le_oob_get_local`. If so that would make it `bt_le_oob_{get|set}_sc_data`.
Similarly it should be `bt_le_oob_sc_data` for the struct